src/about/about.tsx

```diff
@@ -0,0 +1,29 @@
+import { Pokemon } from "../types";
+import { GiWeight } from "react-icons/gi";
+import { TfiRuler } from "react-icons/tfi";
+
+type AboutProp = {
+  pokemon: Pokemon;
+};
+
+const About = ({ pokemon: { weight, height, stats } }: AboutProp) => {
```

Comment:
Você está usando o stats nesse componente?
```suggestion
const About = ({ pokemon: { weight, height } }: AboutProp) => {
```